go.mod

```diff
@@ -46,8 +46,8 @@ require (
 	gonum.org/v1/gonum v0.14.0
 	google.golang.org/protobuf v1.31.0
 	helm.sh/helm/v3 v3.11.1
-	istio.io/api v1.19.3
-	istio.io/client-go v1.19.3
+	istio.io/api v1.19.7-0.20240110022508-68cf4097ffc5
+	istio.io/client-go v1.19.7-0.20240110023605-ad5ce1f48b56
```

Comment:
Can't `v1.19.7` be used here? cc @ialidzhikov

Reply:
Unfortunately, there is no v1.19.7 tag.
See my comment https://github.com/istio/api/issues/2178#issuecomment-1961329408.

Reply:
This will be fixed again in istio. I see the istio maintainer did new changes to fix it again: https://github.com/istio/test-infra/pull/5227. With the new istio version, I hope again to be able to switch to semver tags.